### DISCUSSION TITLE
Removing space after leading diamond

### DISCUSSION THREAD

**Discussion by Kitsunejasutin** [JanDeDobbeleer/oh-my-posh#2570] [Troubleshoot] [answered]
Both of my after leading diamond has this weird line or space. After searching for fix I can't the correct fix in my case. Insight will be great. 

Name

![image](https://user-images.githubusercontent.com/73803767/180912717-c9febbfe-5f07-49b3-8f48-a50ed39ab14b.png)

Right Time dialog

![image](https://user-images.githubusercontent.com/73803767/180912925-fac55b78-39a1-43b0-9015-72eda9fe32a6.png)

my json follows:

      "alignment": "left",
      "segments": [
        {
          "background": "#0077c2",
          "foreground": "#ffffff",
          "leading_diamond": "\u256d\u2500\ue0b6",
          "style": "diamond",
          "template": " {{ .Name }} ",
          "type": "shell"
        },

Right Time dialog

    {
      "alignment": "right",
      "segments": [
        {
          "background": "#40c4ff",
          "foreground": "#ffffff",
          "invert_powerline": true,
          "leading_diamond": " \ue0b6",
          "style": "diamond",
          "template": " \uf5ef {{ .CurrentDate | date .Format }}",
          "trailing_diamond": "\ue0b4",
          "type": "time"
        }
      ],
      "type": "prompt"
    },

**Comment by JanDeDobbeleer** [comment, marked as answer]
@Kitsunejasutin that's the terminal who's failing at correctly printing the character. There's no fix from oh-my-posh to resolve this as that's the terminal's responsibility. Either changing the font, the render engine (if possible) and worst case the terminal ;-)

**Reply by Kitsunejasutin** [reply]
Oh, that's kinda sad. I'll try working around with font. Thanks!

**Comment by Kitsunejasutin** [comment]
I just want to share about this problem, looks like it's the zoom or dpi thing that affects the leading diamond, I noticed that the diamond looks fine on VSCode Terminal 
![image](https://user-images.githubusercontent.com/73803767/195565599-541eeb62-fa90-4c65-9a0f-feb7116d3934.png)

Where the terminal app shows like the picture I shown above. Adjusting the zoom level of it fixed it
![image](https://user-images.githubusercontent.com/73803767/195565463-dd67252c-0412-4dbc-b1ce-748a56ca0e20.png)

**Reply by JanDeDobbeleer** [reply]
@Kitsunejasutin yes, that's a font or terminal issue. I noticed that you need to find the right setting for font size. Nothing we can do about that on our end.
